api/notification/travel.py

```diff
@@ -23,7 +24,7 @@ def notify(self, user_id: str, travel: Travel, event: Event):
         if time_left - travel.duration < 0:
             LOG.info(f'Not enough time to get to event {event}')
         elif 0 < time_left - travel.duration < self.time_delta:
-            response = self.channel.send(message)
+            response = self.channel.send('', message.serialize())
```

Comment:
Just wondering if we can set default `topic=''` so the method signature will be:

```python
def send(message, topic=''):
    pass
```

Reply:
And we can easily invoke:

```python
# self.topic = 'xxx'
send(message)  # msg sent to `xxx`

send(message, '123')  # msg sent to `xxx.123`
```

Reply:
I thought about it but decided to leave it as it is. It would require changes in all repositories where the method is used.

Reply:
k sure ;D